src/python/setup.py.in

```diff
@@ -27,7 +27,7 @@ extra_compile_args=[@GUDHI_PYTHON_EXTRA_COMPILE_ARGS@]
 extra_link_args=[@GUDHI_PYTHON_EXTRA_LINK_ARGS@]
 libraries=[@GUDHI_PYTHON_LIBRARIES@]
 library_dirs=[@GUDHI_PYTHON_LIBRARY_DIRS@]
-include_dirs = [numpy_get_include(), '@CMAKE_CURRENT_SOURCE_DIR@/gudhi/', @GUDHI_PYTHON_INCLUDE_DIRS@]
+include_dirs = [numpy_get_include(), '@CMAKE_CURRENT_SOURCE_DIR@/gudhi/', '@HERA_INCLUDE_DIR@', @GUDHI_PYTHON_INCLUDE_DIRS@]
```

Comment:
By doing I have 2 times hera include dir in `setup.py`. Maybe because you did `include_directories(${HERA_INCLUDE_DIR})` in `src/cmake/modules/GUDHI_submodules.cmake`.

You can remove this modification, I think. I know that we ave a lot of other repeated include_dirs here :sweat:

Reply:
Ah, indeed, I had missed the loop on INCLUDE_DIRECTORIES in CMakeLists.txt.

Reapeated include_dirs are not so bad, what I really don't like is that we end up passing ` -I/usr/include` and `-I/usr/include/x86_64-linux-gnu` to the compiler, which is bad and can break things, but filtering out those directories is really up to CMake. I think it already filters out /usr/include, but too late for us, and last I checked it didn't handle /usr/include/x86_64-linux-gnu specially. Anyway, not this PR.